ecosystem/indexer-grpc/indexer-grpc-utils/src/file_store_operator/local.rs

```diff
@@ -209,4 +209,23 @@ impl FileStoreOperator for LocalFileStoreOperator {
 
         Ok(())
     }
+
+    async fn get_or_create_verification_metadata(
+        &self,
+        _chain_id: u64,
+    ) -> Result<VerificationMetadata> {
+        anyhow::bail!("Verification is not impelemented for local file store.")
```

Comment:
maybe implement this now or follow up with another PR? this would be good to test esp for the post-processor

Reply:
sure!